x-pack/plugins/observability_solution/profiling/public/plugin.ts

```diff
@@ -64,29 +65,47 @@ export class ProfilingPlugin
     ];
 
     const kuerySubject = new BehaviorSubject<string>('');
+    const appUpdater$ = new BehaviorSubject<AppUpdater>(() => ({}));
 
     const section$ = combineLatest([from(coreSetup.getStartServices()), kuerySubject]).pipe(
       map(([[coreStart], kuery]) => {
         if (coreStart.application.capabilities.profiling.show) {
-          const sections: NavigationSection[] = [
-            {
-              label: i18n.translate('xpack.profiling.navigation.sectionLabel', {
-                defaultMessage: 'Universal Profiling',
-              }),
-              entries: links.map((link) => {
-                return {
-                  app: OBLT_PROFILING_APP_ID,
-                  label: link.title,
-                  path: `${link.path}?kuery=${kuery ?? ''}`,
-                  matchPath: (path) => {
-                    return path.startsWith(link.path);
-                  },
-                };
-              }),
-              sortKey: 700,
-            },
-          ];
-          return sections;
+          let isSidebarEnabled = true;
+          coreStart.chrome
```

Comment:
Should this subscription be inside this map? This is creating N subscriptions. isn't it?

Reply:
Rxjs `map()` operator is called once when observable reads data stream. This isn't iterating over anything. Nothing wrong for me here.

Reply:
ah,  yeah. You're right. I hadn't paid attention that this as rxjs map.